pkg/util/sys/ioutils.go

```diff
@@ -127,7 +127,7 @@ func Copy(src string, dst string) error {
     src = filepath.Clean(src)
     dst = filepath.Clean(dst)
     if !Exists(src) {
-        return errors.Stringf("source path [%s] does not exist", src)
+        return errors.New("source path [" + src + "] does not exist")
```

Comment:
We should use `util.Error` here

Reply:
Cyclic imports, was too lazy to find a way around it